modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/service/test/DDMStructureLocalServiceTest.java

```diff
@@ -1194,6 +1335,32 @@ private DDMStructure _updateStructure(DDMStructure structure, String name)
 			ServiceContextTestUtil.getServiceContext(group.getGroupId()));
 	}
 
+	private DDMStructure _updateStructure(
+			String externalReferenceCode, long classNameId, long groupId,
+			String name, DDMStructure structure)
+		throws Exception {
```

Comment:
Can you sort it by following the order of the LocalService methods?

Reply:
In this case, as there is no method explicitly with  ERC, could I maintain the same pattern as the one I used to the add method? In addition, there is no public update method passing the ddmStructure itself.

My method:
```
private DDMStructure _updateStructure(
			String externalReferenceCode, long groupId, long classNameId,
			String name, DDMStructure structure)
```
			
			Add method on LocalService:
		```
	public DDMStructure addStructure(
			String externalReferenceCode, long userId, long groupId,
			long parentStructureId, long classNameId, String structureKey,
			Map<Locale, String> nameMap, Map<Locale, String> descriptionMap,
			String definition, String storageType,
			ServiceContext serviceContext)
```